beetsplug/gmusic.py

```diff
@@ -62,6 +68,14 @@ def upload(self, lib, opts, args):
         self.m.upload(filepaths=files)
         ui.print_(u'Your files were successfully added to library')
 
+    def autoupload(self, session, task):
+        items = task.imported_items()
+        files = [x.path.decode('utf-8') for x in items]
+        ui.print_(u'Uploading your files to Google Play Music...')
+        self.m.upload(filepaths=files)
+        ui.print_(u'Your files were successfully added to your '
+                  + 'Google Play Music library')
```

Comment:
Using `print_` during the import process is likely to interrupt some interactive prompts and such. It’s probably best to use [the plugin’s log](https://beets.readthedocs.io/en/v1.4.7/dev/plugins.html#logging) to emit this output while hiding it by default.